{"name": "test_json_file", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../data/test.json'", "trace": "def test_json_file():\n        print(os.getcwd())\n>       with open('../data/test.json', 'r') as fp:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../data/test.json'\n\ntest/test_file.py:10: FileNotFoundError"}, "attachments": [{"name": "stdout", "source": "e7001c0e-4bff-4da9-86f4-c3ad7f25e549-attachment.txt", "type": "text/plain"}], "start": 1618404976618, "stop": 1618404976619, "uuid": "4fc8cd1c-1d07-456a-a98b-c01e8033c470", "historyId": "d50ca8f5466892ba3c198b58ff77ce88", "testCaseId": "052bea2877096ba85b4b1b5378880999", "fullName": "test.test_file#test_json_file", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_file"}, {"name": "host", "value": "avivatangdeMacBook-Pro.local"}, {"name": "thread", "value": "17344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_file"}]}